source/auth.js

```diff
@@ -1,9 +1,9 @@
 "use strict";
 
 function generateBasicAuthHeader(username, password) {
-    return "Basic " + (new Buffer(username + ":" + password)).toString("base64");
+  return "Basic " + new Buffer(username + ":" + password).toString("base64");
```

Comment:
I think having the braces around the `new Buffer()` is much nicer. `new Something().blah()` is ugly and misleading.